### ISSUE TITLE
Create database schema for shopcarts service

### ISSUE THREAD

**Issue by momo100110** [CSCI-GA-2820-SU23-001/shopcarts#3]
**As a** developer
**I need** to design a database to store information related to micro services for shopcarts
**So that** it is possible to implement CRUD and List APIs
   
 ### Details and Assumptions
 * Using PostgreSQL Database Management System
 * Design the tables containing  necessary attributes related to shopcarts service
 * The two tabes and related attributes are shown below:

**Shopcart Table**
customer_id (primary key), customer_name, shopcart_quantity, shopcart_price
**Item Table**
customer_id (primary key), item_id (primary key), item_name, item_price, item_quantity
   
 ### Acceptance Criteria  
   
 ```gherkin
Given the chosen database management system - PostgreSQL
When I create tables containing attributes related to shopcarts service
Then information can be stored and make APIs achievable
```

**Comment by wl2484** [comment]
After 2nd thought, splitting this story into two, e.g. create another story for choosing a suitable DBMS and preparing corresponding scripts to create the infrastructure, may be more efficient during execution 😄 (because the two stories can be done by different people in parallel), how do you think?

**Comment by wl2484** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/shopcarts-su23-001-6481b88b34844b6416f71850/issues/CSCI-GA-2820-SU23-001/shopcarts/3?planning-poker) with Zenhub @LillisLuo @momo100110 @nijya @wl2484

**Comment by LillisLuo** [comment]
Hi Wan-Yu, you mentioned about preparing corresponding scripts, do you mean the "Create xxx" command? I agree with you that this work can be split into two, one for design the attributes of shopcarts table, the other one for the implementation of the table in a chosen DBMS.

**Comment by momo100110** [comment]
Hi Wan-Yu and Yuzhao, great suggestions! We can split this work as you mentioned. I'm also a little confused about preparing corresponding scripts. Can you explain it in detail or maybe we can discuss it in today's Zoom meeting?

**Comment by wl2484** [comment]
Hi @LillisLuo and @momo100110, sorry for my belated reply. I haven't finished studying the corresponding materials, however, it should be related to [the slide deck: 03-Repeatable-Dev-Environments](https://brightspace.nyu.edu/d2l/le/lessons/290206/topics/8666765). We will need to revise the [docker-compose.yml](https://github.com/CSCI-GA-2820-SU23-001/shopcarts/blob/master/.devcontainer/docker-compose.yml#L25) and probably also others in the [.devcontainer](https://github.com/CSCI-GA-2820-SU23-001/shopcarts/tree/master/.devcontainer) folder to create the database.

**Comment by LillisLuo** [comment]
Here's the initial design of two tables for shopcarts 
<img width="617" alt="tables" src="https://github.com/CSCI-GA-2820-SU23-001/shopcarts/assets/113263069/f215e04d-ef7a-4706-a675-c996be4f3030">
